rad/src/types/mod.rs

```diff
@@ -68,6 +70,7 @@ impl RadonTypes {
             RadonTypes::Float(_) => RadonFloat::radon_type_name(),
             RadonTypes::Integer(_) => RadonInteger::radon_type_name(),
             RadonTypes::Map(_) => RadonMap::radon_type_name(),
+            RadonTypes::RadonError(_) => String::from("RadonError"),
```

Comment:
Why is this not using `radon_type_name`?
```suggestion
            RadonTypes::RadonError(_) => RadonError::radon_type_name(),
```

Reply:
Because `RadonError` is does (and shall not) not implement the `RadonType` trait.

Reply:
A comment in this line would be nice. And if that's important we can use the https://github.com/nvzqz/static-assertions-rs crate to ensure nobody accidentally implements it in the future.